### PR TITLE
[enterprise-4.12] Fixing upgrade section in 4.12 multi-arch docs

### PR DESCRIPTION
**Pull request by kelbrown20**
Version: 4.12
Issue brought up in slack thread 

Preview: 

- [Configuring multi-architecture compute machines on an OpenShift Container Platform cluster -> Upgrading a cluster with multi-architecture compute machines](https://55865--docspreview.netlify.app/openshift-enterprise/latest/post_installation_configuration/multi-architecture-configuration.html#multi-architecture-upgrade-mirrors_multi-architecture-configuration)

- [x] QE approval

### CODE REVIEW

**Comment by kelbrown20** [comment]
@Prashanth684 Can you PTAL at this PR as well when you get the chance, thank you!

File: modules/multi-architecture-upgrade-mirrors.adoc [hunk 1]
```diff
@@ -7,17 +7,4 @@
...
-.Procedure
-* To manually upgrade your cluster, use the following command: 
-[source, terminal]
-+
-----
-$ oc adm upgrade --allow-explicit-upgrade --to-image <image-pullspec> <1>
-----
-<1> You can access the `image-pullspec` object from the link:https://mirror.openshift.com/pub/openshift-v4/multi/clients/ocp/latest[mixed-arch mirrors page] in the `release.txt` file.
\ No newline at end of file
+To upgrade your cluster with multi-architecture compute machines, you must use the `candidate-{product-version}` update channel. For more information, see "Understanding upgrade channels".
```
Comment (Prashanth684) [inline modules/multi-architecture-upgrade-mirrors.adoc:10]: we need to mention that there are two different things that can be done here
- migrate from a single arch to a multi-arch payload
- upgrade within the multi-arch payload


we need to mention that the user needs to change the channel first using `oc adm upgrade channel` and then if the operation is a migration, do an `oc adm upgrade --to-multi-arch`  and if it is an upgrade, do an `oc adm upgrade`. let me verify these steps and make sure they work as expected

Comment (Prashanth684) [reply modules/multi-architecture-upgrade-mirrors.adoc:10]: i just remembered that migration is only supported from 4.13+. i'll get back to you on this

File: modules/multi-architecture-upgrade-mirrors.adoc [hunk 2]
```diff
@@ -7,7 +7,14 @@
 
 = Upgrading a cluster with multi-architecture compute machines
 
-You must perform an explicit upgrade command to upgrade your existing cluster to a cluster that supports multi-architecture compute machines.
+To upgrade your cluster with multi-architecture compute machines, you must use the `candidate-{product-version}` update channel. For more information, see "Understanding upgrade channels".
+
+[NOTE]
+==== 
+You can only upgrade with the `candidate-{product-version}` channel when updating a {product-version} cluster that already uses the multi-architecture payload.
```
[the remark this answers is not in the document]
Comment (kelbrown20) [reply modules/multi-architecture-upgrade-mirrors.adoc:14]: Hmm, I think the 4.12 payload is implied when it says `when updating a 4.12 cluster`

**Comment by kelbrown20** [comment]
@wking Hey Trevor! Could you take a look at this PR that changes the update section in 4.12 for multi-arch compute machines?

File: modules/multi-architecture-upgrade-mirrors.adoc (hunk 2 above)
[the remark this answers is not in the document]
Comment (Prashanth684) [reply modules/multi-architecture-upgrade-mirrors.adoc:14]: ok right that makes sense

**Review by wking — APPROVED** [review]
/lgtm

File: modules/multi-architecture-upgrade-mirrors.adoc [hunk 3]
```diff
@@ -7,7 +7,14 @@
...
 
-You must perform an explicit upgrade command to upgrade your existing cluster to a cluster that supports multi-architecture compute machines.
+To upgrade your cluster with multi-architecture compute machines, use the `candidate-{product-version}` update channel. For more information, see "Understanding upgrade channels".
+
+[NOTE]
+==== 
+Only {product-title} clusters that are already using a multi-architecture payload can update with the `candidate-{product-version}` channel.
+====
+
+If you want to upgrade an existing cluster to support multi-architecture compute machines, you can perform an explicit upgrade command that updates your current single-architecture cluster to a cluster that uses the multi-architecture payload.
```
Comment (mburke5678) [inline modules/multi-architecture-upgrade-mirrors.adoc:17]: s/perform an explicit upgrade command/perform an explicit upgrade command, as shown in the following procedure, ?? Assuming that is what the procedure if for....

**Pull request merged**